packages/wallets/src/features/cfd/screens/Success/Success.tsx

```diff
@@ -58,7 +58,7 @@ const Success: React.FC<TSuccessProps> = ({
                     platform={platform}
                 />
                 <WalletText size='2xs'>
-                    {marketTypeTitle} {!isDemo && `(${landingCompanyName})`}
+                    MT5 {marketTypeTitle} {!isDemo && `(${landingCompanyName})`}
```

Comment:
@thisyahlen-deriv how about using the config file for these hardocded names?
MT5
DerivX
cTraders